Index: Nominatim-2.5.1/utils/setup.php
===================================================================
--- Nominatim-2.5.1.orig/utils/setup.php
+++ Nominatim-2.5.1/utils/setup.php
@@ -93,18 +93,6 @@
 
 	$fPostgisVersion = (float) CONST_Postgis_Version;
 
-	if ($aCMDResult['create-db'] || $aCMDResult['all'])
-	{
-		echo "Create DB\n";
-		$bDidSomething = true;
-		$oDB = DB::connect(CONST_Database_DSN, false);
-		if (!PEAR::isError($oDB))
-		{
-			fail('database already exists ('.CONST_Database_DSN.')');
-		}
-		passthruCheckReturn('createdb -E UTF-8 -p '.$aDSNInfo['port'].' '.$aDSNInfo['database']);
-	}
-
 	if ($aCMDResult['setup-db'] || $aCMDResult['all'])
 	{
 		echo "Setup DB\n";
